server/handlers/user.go

```diff
@@ -103,3 +103,34 @@ func (s *Server) loginImplicitly(w http.ResponseWriter, res *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	json.NewEncoder(w).Encode(user)
 }
+
+func (s *Server) editProfile(w http.ResponseWriter, req *http.Request) {
+
+	verifiedToken := req.Context().Value(ContextKey).(*jwt.Token)
+
+	subject, tokenErr := verifiedToken.Claims.GetSubject()
+	if tokenErr != nil {
+		//not sure what error msg is best here
+		respondWithError(w, "Invalid Token", http.StatusBadRequest)
```

Comment:
Since it's an invalid authorization token, maybe we should send back a status code of unauthorized?